src/libs/auth.ts

```diff
@@ -226,10 +163,11 @@ export type AuthTokenState = AuthTokenSuccessState | AuthTokenExpiredState | Aut
  * When token is successfully loaded, returns an AuthTokenSuccessState.
  * When token fails to load because of an expired refresh token, returns an AuthTokenExpiredState
  * When tokens fails to load because of an error, returns an AuthTokenErrorState
- * @param includeBillingScope
- * @param popUp whether signIn is attempted with a popup, or silently in the background.
+ * @param args whether signIn is attempted with a popup, or silently in the background.
```

Comment:
This comment is no longer accurate. Comments for each argument could be moved to the `OIDCSignInArgs` type definition.